js/index.js

```diff
@@ -1 +1,62 @@
 // Your code goes here
+
+// PREVENTING NAV LINKS FROM RELOADING PAGE
+const nav = document.querySelector(".nav");
+
+nav.addEventListener('click', function(e) {
+    e.preventDefault();
+    console.log("nav link clicked");}
+    );
+
+
+// Flips the logo by clicking on it
+const navLogoHeading = document.querySelector('.logo-heading');
+
+navLogoHeading.addEventListener('dblclick', function (e) {
+    e.target.style.transform = "rotate(180deg)";});
+
+// Changes layout by dragging image
+const s = document.querySelector('.content-section');
+
+s.addEventListener('dragend', function () {
+    s.classList.toggle('swap');});
```

Comment:
So, I wanted to share this with ya to show how you might implement draggable elements on the page:
```js
(() => {
  let element;
  /* events fired on the draggable target */
  document.addEventListener('drag', function (event) {
  });

   document.addEventListener('dragstart', function (event) {
    // store a ref. on the dragged elem
    element = event.target;
    // make it half transparent
    event.target.style.opacity = 0.5;
  });

   document.addEventListener('dragend', function (event) {
    // reset the transparency
    event.target.style.opacity = '';
  });

   /* events fired on the drop targets */
  document.addEventListener('dragover', function (event) {
    // prevent default to allow drop
    event.preventDefault();
  });

   document.addEventListener('dragenter', function (event) {
    // highlight potential drop target when the draggable element enters it
    event.target.style.background = 'purple';
  });

   document.addEventListener('dragleave', function (event) {
    // reset background of potential drop target when the draggable element leaves it
    event.target.style.background = '';
  });

   document.addEventListener('drop', function (event) {
    // prevent default action (open as link for some elements)
    event.preventDefault();
    // move dragged elem to the selected drop target
    event.target.style.background = '';
    element.parentNode.removeChild(element);
    event.target.appendChild(element);
  });
})();
```